docs/android/v2/plugins/Virtual-background.mdx

```diff
@@ -0,0 +1,286 @@
+    }
+```
+## Interfaces
+
+### Instantiate VirtualBackground
+
+Instantiate the 100ms Virtual Background plugin like this:
+
+<Tabs id="instanstiate-VirtualBackground" items={['Kotlin', 'Java']} />
+
+<Tab id='instanstiate-VirtualBackground-0'>
+
+```kotlin
+val hmsSdk = HMSSDK
+      .Builder(application)
+      .build()
+val virtualBackground = HMSVirtualBackground(hmsSdk, imageBitmap)
+```
+
+</Tab>
+
+<Tab id='instanstiate-VirtualBackground-1'>
+
+```java
+HMSSDK hmsSdk = new HMSSDK
+                       .Builder(application)
+                       .build();
+HMSVirtualBackground virtualBackground = new HMSVirtualBackground(hmsSdk, imageBitmap)
+```
+</Tab>
+
+That's it. You have instantiated Virtual Background plugin successfully. 🥳
+
+Now let's take a look at the signature of the HMSVirtualBackground.
+
+`HMSVirtualBackground` accepts 2 arguments -
+-   `hmsSdk`: an object of type `HMSSDK` class
+-   `imageBitmap`: A background image bitmap you want to replace your background to.
+
+### Check if plugin is supported
+
+`HMSVirtualBackground.isSupported` can be used to check if the API level/Resolution of device is supported or not.
+It will return True/False based on these parameters.
+
+<Tabs id="Check-plugin-support" items={['Kotlin', 'Java']} />
+
+<Tab id='Check-plugin-support-0'>
+
+```kotlin
+if(virtualBackground.isSupported()){
+    console.log("Device/Resolution is supported");
+}else{
+    console.log("Device/Resolution is not supported, check Supported Devices/Versions/Resolutions section");
+}
+```
+</Tab>
+
+<Tab id='Check-plugin-support-1'>
+
+```java
+if(virtualBackground.isSupported()){
+    console.log("Device/Resolution is supported");
+}else{
+    console.log("Device/Resolution is not supported, check Supported Devices/Versions/Resolutions sections");
+}
+```
+</Tab>
+
+
+### Init(Optional)
+
+Init is used to load the machine learning model of Virtual Background for the first time.
+It takes on an average 50-200 ms.<br/>
+Calling init is handled internally by SDK if not done by user, in this case addPlugin call will
+take 50-200 ms for the first time and then less than 10ms in the subsequent calls.
+Check [this section](#add-plugin) for addPlugin API usage
+
+<Tabs id="Init" items={['Kotlin', 'Java']} />
+
+<Tab id='Init-0'>
+
+```kotlin
+try{
+   virtualBackground.init();
+}catch(err){
+   console.log("failed to init Plugin", err);
+}
+```
+</Tab>
+
+<Tab id='Init-1'>
+
+```java
+try{
+   virtualBackground.init();
+}catch(err){
+   console.log("failed to init Plugin", err);
+}
+```
+</Tab>
+
+
+### Change Background
+
+Call `setBackground` on the `HMSVirtualBackground` instance to update the background again later if required.
+It accepts imageBitmap or String as a parameter
+
+-  `imageBitmap`- Bitmap - A background image bitmap you want to replace your background to.
+-  `None` - String - This will remove the background effect from the video
+
+Image will get fit to the video by maintaining the aspect ratio.
```

Comment:
Image will be scaled to fit to the video. Scaling will the maintain image aspect ratio.